### ISSUE TITLE
Add a web app manifest

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#10214]
See eg https://developers.google.com/web/updates/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android?hl=en

* This will show the specified icon and name when adding it to the home screen. 

![homescreen](https://cloud.githubusercontent.com/assets/273120/15813803/31afbe44-2bc2-11e6-9adb-6038fc55e2bd.jpg)

* It will show a splash screen while loading Piwik initially:

![screenshot_2016-06-06-08-33-58](https://cloud.githubusercontent.com/assets/273120/15813805/361aed3c-2bc2-11e6-8f12-8fab9712d29e.png)

* It will launch the browser without header so you can actually see more of the content.

Example definition: 

```json
{
  "short_name": "Piwik",
  "name": "Piwik - Open Source Analytics",
  "icons": [
    {
      "src": "../images/favicon.png",
      "sizes": "32x32",
      "type": "image/png"
    }
  ],
  "start_url": "../../../",
  "display": "standalone",
  "orientation": "portrait"
}
```

We could change some colors but not needed for now I'd say. Specifying a web app manifest theme color might be useful later. 

@mattab can you send me the favicon in a higher resolution? Eg up to 192x192

We should not add manifest if a custom logo is specified.

**Comment by mattab** [comment]
That's again something new and great. 

Here is the large res favicon, freshly created from the SVG logo:

![favicon-large](https://cloud.githubusercontent.com/assets/466765/16672694/dc7ca06a-44fb-11e6-971b-1a772fd81841.png)

Looking forward to using our Piwik web app as if it was a mobile app!

**Comment by mattab** [comment]
hoping @tsteur we could maybe get this done in LTS? it is an awesome feature :open_mouth:

**Comment by tsteur** [comment]
done. Although I think it's not as relevant for 2.X as it is not as good on mobile anyway :)

We could later add some more favicon metadata for iOS devices etc but this issue should be closed once the PR is merged
